src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -5006,7 +5003,7 @@ namespace WindowEquivalentLayer {
         }
     }
 
-    bool ASHWAT_Thermal(CFSTY const &FS,  // fenestration system
+    bool ASHWAT_Thermal(CFSTY &FS,        // fenestration system
```

Comment:
Alright, another removal of const, and I see that below in this function you are assigning the value of the error index.  That's fine, but what I can't see (from this diff) is why the `FS` variable can't be constant.  It looks like the first argument being passed into here is called `CFS(EQLNum)`, not `FS` in the function above.  I think I'm still just missing something because the diff is only showing part of the story.

Reply:
The only reason that I removed the constant was I cannot update the error index variable (WEQLSolverErrorIndex) that I added for this fix for reporting recurring errors. If I can pass as const and still be able to update the error index variable, I am happy to to do it.